autoload/fireplace.vim

```diff
@@ -1798,7 +1798,7 @@ endfunction
 
 function! fireplace#info(symbol) abort
   if fireplace#op_available('info')
-    let response = fireplace#message({'op': 'info', 'symbol': a:symbol}, v:t_dict)
+    let response = fireplace#message({'op': 'info', 'symbol': a:symbol, 'ns': fireplace#ns()}, v:t_dict)
```

Comment:
You sure this is necessary? `fireplace#message()`already does the effectively the same with `'ns': v:true`.

Reply:
You're right, this is redundant (I've removed this change)